infra/web-app/ingress.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     traefik.ingress.kubernetes.io/router.middlewares: olfyi-redirect@kubernetescrd
 spec:
   rules:
-    - host: dev.0l.fyi
+    - host: dev.scan.openlibra.world
```

Comment:
Current canonical host name is: scan.openlibra.io. However, it isn't actually deployed from these files (see here: https://github.com/whichnode/libra-explorer-deployment) so whatever is in this file is moot anyway.